platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/ConfigurationFileProxy.java

```diff
@@ -69,46 +75,29 @@ public ConfigurationFileProxy(ConfigurationStore configurationStore) {
         }
     }
 
-    protected BundleContext getContext() {
-        Bundle cxfBundle = FrameworkUtil.getBundle(ConfigurationFileProxy.class);
-        if (cxfBundle != null) {
-            return cxfBundle.getBundleContext();
-        }
-        return null;
+    public void writeSolrConfiguration(String core) {
```

Comment:
Any specific reason why this isn't done in an `init()` method instead?

If it needs to be kept, we should add a JavaDoc comment.